frontend/components/forms/fields/SelectTargetsDropdown/_styles.scss

```diff
@@ -107,7 +107,7 @@
 
       .Select-value {
         line-height: 34px;
-        border: 1px solid $core-dark-blue-grey;
+        border: 1px solid $ui-fleet-black-75;
```

Comment:
I'm not seeing changes to the select targets dropdown included in the specs for either ticket. Should it be? Not sure how to check this.

Reply:
@lukeheath 
I removed all $core-dark-blue-grey in this PR and replaced them with $ui-fleet-black-75 per
- the styling doc:
https://docs.google.com/document/d/1x9E6fSzVSYwuMy0TQoRQUA8eMhZO09y4LakjCRZSHjo/edit?disco=AAAAi5X7Fas
- gabe's ticket related to the styling doc #8385 "Change colour on the page text description in the page header (xs)"